src/vcf/meta_entry.cpp

```diff
@@ -214,6 +214,13 @@ namespace ebi
             value["Type"] != "String") {
             throw new MetaSectionError{entry.line, "INFO metadata Type is not a Integer, Float, Flag, Character or String"};
         }
+
+        // Check for predefined tag AA
+        if (value["ID"] == "AA") {
+            if (value["Type"] != "String") {
```

Comment:
Oh this idea is great, but what about Number? I guess that would be required for a few other tags, shouldn't I include it?